src/tower_arduino_ide/relay_bus_defines.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RELAY_BUS_DEFINES_H
```

Comment:
> This is kinda weird... The way your data is formatted isn't very clear and it's hard to sanity check/extend later. 

Continuing on the above: Are you assigning meaning to specific bits? You should clarify your message formatting and explain why it's done that way.

Reply:
Will do in a future commit :)